Dockerfile

```diff
@@ -14,11 +15,12 @@ ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true
 # Since 0.14.0 release of html-template chromium is needed for pdf generation.
 # More custom packages for specific template should not be added to this dockerfile. Instead, we should come up with some extensibility solution.
 RUN apk --update add git chromium && \
+    apk add --no-cache --virtual .gyp python3 make g++ && \
     rm -rf /var/lib/apt/lists/* && \
     rm /var/cache/apk/*
 
 # Installing latest released npm package
-RUN npm install --ignore-scripts -g @asyncapi/cli@"$ASYNCAPI_CLI_VERSION"
+RUN npm install --ignore-scripts -g @asyncapi/cli
```

Comment:
@eelcofolkertsma you added `ASYNCAPI_CLI_VERSION` back in line 4 but it is still missing here - basically not used

Reply:
OK, fixed that one too
If you use Dockerfile without change you get latest version of cli. If you add a version-value you get image for that version of cli
Tested OK in Github codespaces (default version=1.7.3, forced version=1.7.2) with docker build -t cli .  and docker run cli --version